src/main/java/Duke/UI.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * UI for Duke.
+ */
+public class UI {
+    private final Scanner myScanner;
+
+    public UI() {
+        this.myScanner = new Scanner(System.in);
+    }
+    /**
+     * Prints greeting text of the robotFriend.
+     */
+    public static String greet() {
+        return "Hello I am your RobotFriend. How can i help you?";
+    }
+
+
+    /**
+     * Prints the exiting text in a robot way.
```

Comment:
Since you are returning a String, you might want to add @return in your javadoc to describe what the function is returning.